src/Table/Table.tsx

```diff
@@ -179,8 +239,8 @@ const ActionColumn: React.FunctionComponent<ActionColumnProps> = (props: ActionC
                 >
                     {props.primaryActionButton.label}
                 </button>
-            }
-            {props.actionLinks && props.actionLinks.length &&
+            )}
+            {props.actionLinks && props.actionLinks?.length && (
```

Comment:
I think if you test this with an empty action links array it will actually render `0` here. It's better use:
```tsx
{props.actionLinks && props.actionLinks?.length ? (
    /* your code */
) : null }
```